dango/vesting/src/state.rs

```diff
@@ -0,0 +1,16 @@
+use {
+    dango_types::vesting::Position,
+    grug::{Addr, Counter, IndexedMap, MultiIndex},
+};
+
+pub const POSITION_INDEX: Counter<u64> = Counter::new("position_index", 0, 1);
+
+pub const POSITIONS: IndexedMap<u64, Position, PositionIndexes> =
+    IndexedMap::new("positions", PositionIndexes {
+        user: MultiIndex::new(|_, position| position.user, "positions", "positions_user"),
```

Comment:
Let's always use singular for a map's namespace (`position` instead of `positions`) and two underscores for the index (`position__user` instead of `position_user`)